Server/ConcordiaCurriculumManager/DTO/Dossiers/EditDossierDTO.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+namespace ConcordiaCurriculumManager.DTO.Dossiers
+{
+    public class EditDossierDTO
+    {
+        public required Guid Id { get; set; }
+
+        public required Guid InitiatorId { get; set; }

```

Comment:
The InitiatorId doesn't seem used, and it's not something we would want changed in any case, it might be better to remove it from here.

Reply:
Sounds good